cypress/fixtures/kitchenSink.html

```diff
@@ -0,0 +1,94 @@
+<h1 class="markdown__heading">Basic Markdown Testing</h1><p>Tests for text style, code blocks, in-line code and images, lines, block quotes, and headings.</p>
```

Comment:
Could you rename this fixture file to something more descriptive like `longMarkdownPost.html`?